File: JAVA/Count numbers containing 4.java
class Solution {
    public static int countNumberswith4(int n) {
        int cnt=0;
        int i=1;
       while(i<=n)
        {
          int num=i;
          while(num>0)
          {
              int r=num%10;
              if(r==4)
              {
                  cnt++;
                  break;
              }
              num=num/10;
          }
          i++;
        }
        return cnt;
    }
}

File: C++/Count numbers containing 4.cpp
class Solution {
  public:
    int countNumberswith4(int n) {
        int count = 0;
        for (int num = 1; num <= n; num++) {
            if (to_string(num).find('4') != string::npos) {
                count++;
            }
        }
        return count;
    }
};
